ui/src/app/store/script/utils.ts

```diff
@@ -0,0 +1,23 @@
+import type { AnyObject } from "app/base/types";
+
+// The script parameters are parsed from a JSON object with any shape so we have
+// to do a bit more work to be sure we're retrieving valid values.
+export const getParameterObject = (
```

Comment:
I feel like `getObjectValue` and `getObjectString` might be more descriptive function names. I'm never really sure what a "parameter" is?

Reply:
Done, I was trying to capture that these were for use with the `ScriptsParameters` objects, but the generic name is fine.